app/src/androidTest/java/io/left/meshim/database/MigrationTest.java

```diff
@@ -84,4 +85,9 @@ public void migrate2To3() throws IOException {
         SupportSQLiteDatabase db = helper.createDatabase(TEST_DB, 2);
         db = helper.runMigrationsAndValidate(TEST_DB, 3, true, MIGRATION_2_3);
     }
+    @Test
```

Comment:
A comment on here as to what happened in this migration and why we only nee to validate it would be helpful.